src/web/mjs/connectors/KissLove.mjs

```diff
@@ -0,0 +1,13 @@
+import FlatManga from './templates/FlatManga.mjs';
+
+export default class LHScan extends FlatManga {
```

Comment:
Change classname from `LHScan` to `KissLove`, i think there is already a class with that name.